src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -3403,6 +3409,11 @@ public static void parseTownClaimCommand(Player player, String[] split) {
 
 				resident = townyUniverse.getDataSource().getResident(player.getName());
 				town = resident.getTown();
+
+				if (TownySettings.isTownBankruptsyEnabled() && town.isBankrupt()) {
+					throw new TownyException(TownySettings.getLangString("msg_err_bankrupt_town_cannot_claim"));

```

Comment:
Change to `if (town.isBankrupt())`
Language message should not be something relating to siegewar.

Reply:
Done